spec/acceptance/api/proxy_spec.rb

```diff
@@ -6,6 +6,10 @@
 
   let(:service_id) { service.id }
 
+  before do
+    service.build_default_backend_api_config.save!
```

Comment:
I don't get it. Why `save!` on the backend API if the the method already follows the state of the service either calling `.new` or `.create!`?

Reply:
For this: https://github.com/3scale/porta/blob/679f351d2382e677f557c602ad73b7122c905efd/spec/acceptance/api/proxy_spec.rb#L31

This does not follow any api call.